### ISSUE TITLE
Unexpected behaviour with right to left direction

### ISSUE THREAD

**Issue by Abisar** [express-labs/pure-react-carousel#97]
My body tag has `direction: rtl`. If I swipe from left to right, my expected behaviour is I want to go from slide 1 to slide 2 but it tries to go from slide 1 to 0 which doesn't exist.

Is there any support for RTL direction? It's basically an inversion of everything related to x-axis.

**Comment by tim-steele** [comment]
At this time we haven't added support to it. It is something we can look at doing, but we are also open to a pull request.

**Comment by mrbinky3000** [comment]
@Abisar so, when I drag the slide from the left to the right, the slider would animate in the opposite direction from right to left?

Or, are you talking about the ButtonNext and ButtonBack components?  If you are talking about the buttons, simply place ButtonNext on the left side of your carousel and ButtonBack on the right side.

If you are using the Dot component, simply display them from high to low.   

Basically, you can place the buttons "backwards" and simulate rtl behavior.  The only question I have is about inverting the dragging?  I tried to find other carousels that do this.  Do you know of one so I can check it out?

**Comment by akramabdulrahman** [comment]
well I am having the same problem under dir="RTL", if there was an invert option for the drag I mean like 
 drag=invert 
that would just invert the translation animation.
right now in rtl this is what happens 
![image](https://user-images.githubusercontent.com/1550238/101151082-34617900-3632-11eb-85c0-49584ffb1121.png)

I haven't read the lib yet , I need this to be enabled , gonna see if i can check it out . 
shared this to make sure i know what needs to be done . 

-see where the drag event is being made. if the provider has an invertDrag prop with true x will become window.width - x 
if the thing is vertical its window.height -y  

thanks for this carousel O.o

**Comment by akramabdulrahman** [comment]
tried to clone the thing and change things up :D the scripts arent exactly made for windows and such :D this might take a while

**Comment by tim-steele** [comment]
@akramabdulrahman if an invert feature needs to happen, then we are open for a pull request.
